media/main.js

```diff
@@ -79,36 +90,44 @@ function () {
 				const requirementName = json.requirementName;
 				const isAvailable = json.result;
 
+				let requirementMessage = 'Not currently available';
```

Comment:
it's been that way since the beginning and I've been looking at it as the general pattern "X" and "not X" for simplicity so that when you look at the first word in the status you get a quicker indication. so I'm going to leave it. :)